spec/spec_helper_acceptance.rb

```diff
@@ -10,7 +10,7 @@
 install_module_dependencies
 
 # Install aditional modules for soft deps
-install_module_from_forge('puppetlabs-apt', '>= 4.1.0 < 5.0.0') if fact('os.family') == 'Debian'
+install_module_from_forge('puppetlabs-apt', '>= 4.1.0 < 5.1.0') if fact('os.family') == 'Debian'
```

Comment:
can you please update this to `< 6.0.0`?  We always try to support the whole major release range.